### PR TITLE
docs(pages/*): punctuation

### PR DESCRIPTION
**Pull request by jly36963**
Simple docs changes.
PR on main, let me know if that is wrong. ([ref](https://remix.run/docs/en/v1/pages/contributing#branches))

Sorry for being nitpicky, having a copy-editor mindset makes me read more attentively 😂 

(If there is linting on the markdown files, CI might fail. I normally use Deno for formatting, and I turn it off when I'm contributing to other codebases. I'll set up the dev environment properly and fix it if that ends up being the case.)

### CODE REVIEW

File: docs/pages/contributing.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ When you start a pull request, the remix-cla-bot will prompt you to review the C
 This document refers to contributors with the following roles:
 
 - **Admins**: GitHub organization team with admin rights, they set and manage the Roadmap.
-- **Collaborators**: GitHub organization team with write access. They manage issues, prs, discussion, etc.
```
Comment (jly36963) [inline docs/pages/contributing.md:28]: PRS guitars are cool 🎸

File: docs/pages/community.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ You can also subscribe to [releases on GitHub][releases-on-git-hub].
 
 In addition to the [official][official] [tutorials][tutorials], here are Remix tutorials you can follow to learn Remix:
 
-- [How to Debug Remix loaders and actions in VS Code][how-to-debug-remix-loaders-and-actions-in-vs-code] by [Michael Carter][michael-carter].
+- [How to Debug Remix loaders and actions in VS Code][how-to-debug-remix-loaders-and-actions-in-vs-code] by [Michael Carter][michael-carter]
```
Comment (jly36963) [inline docs/pages/community.md:56]: Removed period to be consistent with other bullets

File: docs/pages/contributing.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ This document will familiarize you with our development process as well as how t
 
 ## Contributor License Agreement
 
-All contributors sending Pull Request need to sign the Contributor License Agreement (CLA) that explicitly assigns ownership of the contribution to us.
+All contributors sending a Pull Request need to sign the Contributor License Agreement (CLA) that explicitly assigns ownership of the contribution to us.
```
Comment (jly36963) [inline docs/pages/contributing.md:17]: Indefinite article seemed appropriate here

File: docs/pages/contributing.md [hunk 4]
```diff
@@ -46,16 +46,16 @@ If you have an idea for a new feature, please don't send a Pull Request, but fol
 4. Owners create an **RFC** from the Proposal and development can begin.
 5. Pairing is highly encouraged, particularly at the start.
 
-### Bug Fix Pull Requests
+### Bug-Fix Pull Requests
```
Comment (jly36963) [inline docs/pages/contributing.md:49]: [hyphens with compound modifiers](https://www.grammarly.com/blog/hyphen-with-compound-modifiers/)

Eg: 

Screenshot 2023-01-29 at 10 25 33 PM

**Review by machour — APPROVED** [review]
Thank you;

😄

**Pull request merged**